apple-pie/styles/style.css

```diff
@@ -0,0 +1,66 @@
+.box {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-size: cover;
+    background-position: center; 
```

Comment:
To apply the background image through the `style.css` file you just need to add two lines of code here in the `.box{` selector:

```
  background-image: url(/apple-pie/images/apple-pie.jpg);
  height: 75vh;
```